src/controllers/user.controller.ts

```diff
@@ -156,4 +152,44 @@ export class UserController {
   async deleteById(@param.path.string('id') id: string): Promise<void> {
     await this.userRepository.deleteById(id);
   }
+
+   /**
+ * metodos personalizados para la Api
+ *
+ */
+
+   @post('/identify-users')
+   @response(200, {
+     description: "identificar un usuario por correo y clave",
+     content: {'application/json': {schema: getModelSchemaRef(Credentials)}}
+   })
+   async identifyUser(
+     @requestBody(
+       {
+         content:{
+           'aplication/json':{
+             schema: getModelSchemaRef(Credentials)
+           }
+         }
+       }
+     )
+     credentials : Credentials
+   ): Promise<object> {
+    let user = this.serviceSecurity.identifyUser(credentials);
+    if (user) {
```

Comment:
Change structure if-else.
First evaluates escape condition and "else"  isn´t necesary.

Something like:
if(!user){
    return new HttpErrors[401]("incorrect credentials.");
}


let code2fa = this.serviceSecurity.createTextRandom(5);
      let login: Login = new Login();
      login.userId = user._id!;
      login.code2fa = code2fa;
      login.codeState2fa = false;
      login.token = "";
      login.tokenState = false;
      this.repositoryLogin.create(login);
      // notify the user via mail or sms
      return user;